packages/internal-lib-build/bin/internal-lib-build.js

```diff
@@ -34,6 +39,12 @@ const main = () => {
             `${babel} --config-file ${babelConfig} src -x ".js",".jsx",".ts",".tsx" --ignore "**/test_fixtures/*","*.test.js","test.js" --out-dir dist --copy-files`
         )
         execSync(`node ${prepareDist}`)
+
+        // Conditionally run tsc if a typescript config exists. We typically do this to extract types from our SDK.
+        const productDir = process.cwd().split(p.sep).at(-1)
+        if (USE_TSC.includes(productDir) && fs.existsSync('tsconfig.json')) {
+            execSync(`npx tsc`)
+        }
```

Comment:
if this only runs on exceptions, i.e. not every project runs this, why don't we just add it to the `build` script in that specific package?

```json
{
  "scripts":{
      "build": "internal-lib-build build && tsc"
    }
}
```

Reply:
Thats a good point. I had done this last minute, I think your suggestion is probably the simpler solution. I'll go ahead and try that change and if it works I'll run with it.

Reply:
Worked like a charm 👍